images/restd/slurmrestd_entrypoint.sh

```diff
@@ -2,6 +2,14 @@
 
 set -e # Exit immediately if any command returns a non-zero error code
 
+echo "Link users from jail"
+for file in passwd group shadow gshadow; do
+  rm -f /etc/$file
```

Comment:
In all other images we removed users in Docker file and link them in entrypoint

Reply:
My point is not about splitting... I think we need to do that in one way in all places)